deps/v8/src/log-utils.cc

```diff
@@ -56,9 +56,9 @@ void Log::Initialize(const char* log_file_name) {
 
     if (output_handle_ != nullptr) {
       Log::MessageBuilder msg(this);
-      msg.Append("v8-version,%d,%d,%d,%d,%d", Version::GetMajor(),
+      msg.Append("v8-version,%d,%d,%d,%d,%s,%d", Version::GetMajor(),
```

Comment:
You mean print unconditionally or only when it's a non-empty string?  The former is easier but really, it's up to you.